static/src/js/components/GranuleResults/GranuleDownloadButton.jsx

```diff
@@ -30,6 +32,8 @@ export const GranuleDownloadButton = (props) => {
   } = props
 
   const { disableDatabaseComponents } = getApplicationConfig()
+  const { apiHost } = getEnvironmentConfig()
+  const isLoggedIn = authToken !== ''
```

Comment:
I see we have a similar check in at least one other spot (static/src/js/components/SearchPanels/SearchPanels.jsx). There we check against an empty string or null. I'd consider creating a util to encapsulate this functionality.